optbinning/scorecard/plots.py

```diff
@@ -0,0 +1,289 @@
+"""
+Modules with plots for visualizing model performance.
```

Comment:
Module (remove s)

Reply:
Should I add also your name to the module?